src/Microsoft.Health.Dicom.Core/Features/Model/InstanceProperty.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Model
+{
+    public class InstanceProperties
```

Comment:
nit: File name didnt match the class name. I think InstanceProperty sounds good to me